lobby-db/src/main/resources/db/migration/V2.00.05__drop_legacy_password.sql

```diff
@@ -0,0 +1,2 @@
+alter table lobby_user drop column password;
+alter table lobby_user alter column bcrypt_password set not null;
```

Comment:
Good consideration. I thought it would not have been dropped, but it appears it is dropped. When I try to drop the constraint after the column is dropped, I get a SQL error:

>   ERROR: constraint "lobby_user_pass_check" of relation "lobby_user" does not exist

Reply:
Smart Database